### PR TITLE
Delete mentions of copying order in Overview.md

### PR DESCRIPTION
**Pull request by aheejin**
After the spec change in #126, byte copying order is not observable.

### CODE REVIEW

File: proposals/bulk-memory-operations/Overview.md [hunk 1]
```diff
@@ -281,10 +281,8 @@ Note that it is allowed to use `memory.init` on the same data segment more than
 once.
 
 Initialization takes place bytewise from lower addresses toward higher
```
[the remark this answers is not in the document]
Comment (aheejin) [reply proposals/bulk-memory-operations/Overview.md]: Done

File: proposals/bulk-memory-operations/Overview.md [hunk 2]
```diff
@@ -339,10 +334,9 @@ A trap occurs if:
...
 
-A trap resulting from an access outside the source or target region
-only occurs once the first byte that is outside the source or target
-is reached (in the defined copy order).  Bytes written before the trap
-stay written.
+Memory copying order of bytes is not observable. When a trap resulting from an
+access outside the source or target region occurs, partial copying does not take
+place.
 
 (Data are read and written as-if individual bytes were read and
```
[the remark this answers is not in the document]
Comment (aheejin) [reply proposals/bulk-memory-operations/Overview.md]: Replaced the paragraph with the sentence from the memmove link. Let me know if you think it's not sufficient.

File: proposals/bulk-memory-operations/Overview.md [hunk 3]
```diff
@@ -362,10 +356,8 @@ The instruction has the signature `[i32 i32 i32] -> []`. The parameters are, in
 A trap occurs if:
 * the destination offset plus size is greater than the length of the target memory   
 
-Filling takes place bytewise from lower addresses toward higher
-addresses.  A trap resulting from an access outside the target memory
-only occurs once the first byte that is outside the target is reached.
-Bytes written before the trap stay written.
+Memory writing order of bytes is not obserable. When a trap resulting from an
```
[the remark this answers is not in the document]
Comment (aheejin) [reply proposals/bulk-memory-operations/Overview.md]: Thanks! Done.

**Comment by lars-t-hansen** [comment]
FWIW, I think this partly duplicates and conflicts with #130, which I just merged (things got delayedb because @rossberg was away and then after that I was inattentive).

**Comment by aheejin** [comment]
Oh, I didn't know your PR was pending. I deleted all duplicate paragraphs and this is what remains 😂 PTAL.

**Pull request merged**